api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-# .NET Core API Reference
+# .NET Core API Reference 
 
-These documents cover the API reference for .NET Core. You can use the search function 
-on the left-hand side to search for an API or use the Table of Contents on the left to navigate.
+The .NET Core API Reference documents the types and type members available in .NET Core. You can navigate to the documentation for a particular type (a class, structure, delegate, enum, or interface) or type member (a constructor, method, property, or event) in any of three ways:
 
-If you find an issue or an inconsistency, we would love to hear about it! 
-Go ahead to [.NET Core docs repo](https://github.com/dotnet/core-docs/) On Github and leave 
-an issue to let us know.
+- By using the search function in the left-hand pane to search for the **type** by name. (Search by member name is not supported.)
 
+- By navigating to the documentation for the type by expanding the **namespace** that that type is found in in the left hand pane, and then selecting the type.
```

Comment:
I'd rewrite this sentence to remove the duplicates that that and in in. Maybe **namespace** where that type is found in the left hand pane?